conans/model/info.py

```diff
@@ -380,7 +379,17 @@ def loads(text):
                               raise_unexpected_field=False)
         result = ConanInfo()
         result.invalid = None
-        result.settings = Values.loads(parser.settings)
+
+        def _loads_settings(settings_text):
+            settings_result = []
+            for line in settings_text.splitlines():
+                if not line.strip():
+                    continue
+                name, value = line.split("=", 1)
+                settings_result.append((name.strip(), value.strip()))
+            return settings_result
+
+        result.settings = _loads_settings(parser.settings)
```

Comment:
Shouldn't it be better to have it on `Settings` side? I guess that you cannot use the current `Settings.loads()` but I don't get to understand why.

Reply:
That is a very good question. This is not an actual Settings object. This is a plain list of tuples (string elems) that needs to be loaded from conaninfo.txt to be serialized to json from the server to the client. It is not a full fledged Settings object, and it cannot be one, because for that, it should get the settings.yml file, and that only exists in the client side, whiles this operation happens in the server side too (when a client is doing a search or list of binaries in the server)

This is why I kept in this file. It is about reading conaninfo.txt and serializing it for client consumption (the client side also don't deserialize the settings at all to a ``Settings`` object, it keeps the plain python object.

Reply:
Ok, I got it. Thanks for the explanation! 👍